### ISSUE TITLE
Ability to remove entries from a dictionary

### ISSUE THREAD

**Issue by pgolo** [pgolo/pilsner#6]
Use case is yet unclear, but it is possible that the ability to remove entries from already compiled model might be handy.
Something like:
```python
m = pilsner.Model('model_name') # load compiled model
m.remove_entry('substring we do not want to recognize')
```

**Comment by pgolo** [comment]
As a note: rather than removing, it is quicker to tag trie branch that corresponds to the label desired to be ignored.

**Comment by pgolo** [comment]
Resolved with PR #24
